usage-rules/ecto.md

```diff
@@ -1,3 +1,4 @@
+<!-- phoenix:ecto-start -->
```

Comment:
These shouldn't be included in the file itself, otherwise it will be doubled up when we build the file. You'll want to programmatically do this in your script that copies them.